scripts/enum/item.lua

```diff
@@ -5840,12 +5840,15 @@ xi.item =
     GLEAMING_ZAGHNAL                    = 18963,
     DIRE_SCYTHE                         = 18964,
     TERPSICHORE_75                      = 18989,
+    MURGLEIS_75                         = 18995,
```

Comment:
Could you also please change the entries in item_basic (just the name, not sortname) to match these enums?  This will be vital to add magian support for upgrades

Reply:
all set